selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/keycloak/executor/OpenShiftPodCommandExecutor.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.core.client.keycloak.executor;
+
+import static java.lang.String.format;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.inject.name.Named;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.commons.io.FileUtils;
+import org.eclipse.che.selenium.core.provider.OpenShiftWebConsoleUrlProvider;
+import org.eclipse.che.selenium.core.utils.process.ProcessAgent;
+import org.eclipse.che.selenium.core.utils.process.ProcessAgentException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** @author Dmytro Nochevnov */
+@Singleton
+public class OpenShiftPodCommandExecutor implements KeycloakCommandExecutor {
+  private static final Logger LOG = LoggerFactory.getLogger(OpenShiftPodCommandExecutor.class);
+
+  private static final boolean IS_MAC_OS =
+      System.getProperty("os.name").toLowerCase().startsWith("mac");
+  private static final String DEFAULT_OPENSHIFT_USERNAME = "developer";
```

Comment:
So, this only works if auth isn't configured in the cluster (by default with oc cluster up)? Wouldn't it make more sense to have it in conf, so that you can run tests against  Che deployed on any cluster?

Reply:
yes, it would. Thank you for advice.